integration-tests/cucumber-tests-execution/src/main/java/org/opensmartgridplatform/cucumber/execution/AppBase.java

```diff
@@ -40,14 +44,19 @@ public static int run(final AppBase app, final String[] testClasses, final Strin
             p.parseArgument(args);
             LOGGER.info("Start");
             final int retval = app.runTests(testClasses);
-            LOGGER.info("End. Retval = {}", retval);
+            LOGGER.info(END_RETVAL, retval);
 
             return retval;
         } catch (final CmdLineException e) {
             LOGGER.error("Incorrect usage", e);
             LOGGER.error("java -jar <...>.jar [opts] ...");
             p.printUsage(System.err);
             return -1;
+        } catch (final Throwable e) {
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Catch Exception instead of Throwable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3AS1181)